kofta/src/app/components/LanguageSelector.tsx

```diff
@@ -45,6 +45,7 @@ export const LanguageSelector: React.FC<LanguageSelectorProps> = ({
     { value: "pt-PT", label: "🇵🇹 Português (Europeu)" }, // Portuguese (Portugal)
     { value: "ro", label: "🇷🇴 Română" }, // Romanian
     { value: "ru", label: "🇷🇺 Русский" }, // Russian
+    { value: "si", label: "🇱🇰 සිංහල" },
```

Comment:
I'd add a // comment there with the language.

Reply:
fixed it